client/src/components/landing/index.js

```diff
@@ -1,107 +1,202 @@
 import React, { Fragment } from 'react';
-
-import { Grid, Row, Col } from '@freecodecamp/react-bootstrap';
+import { Grid, Col, Row } from '@freecodecamp/react-bootstrap';
 import Helmet from 'react-helmet';
 import PropTypes from 'prop-types';
-import { Link } from 'gatsby';
-import { uniq, partition } from 'lodash';
+import { graphql, useStaticQuery, Link } from 'gatsby';
+import Img from 'gatsby-image';
+import { uniq } from 'lodash';
+
 import { Spacer } from '../helpers';
 import Login from '../Header/components/Login';
-import CompanyLogos from './components/CompanyLogos';
-import { AsFeatureLogo } from '../../assets/images';
+import {
+  AmazonLogo,
+  AppleLogo,
+  MicrosoftLogo,
+  SpotifyLogo,
+  GoogleLogo,
+  AsFeatureLogo
+} from '../../assets/images/components';
+import testimonialsMeta from './landingMeta';
 
 import './landing.css';
-import '../Map/map.css';
+
+const { dasherize } = require('../../../../utils/slugs');
 
 const propTypes = {
-  nodes: PropTypes.array
+  page: PropTypes.string
 };
 
-const BigCallToAction = () => (
-  <Row>
-    <Col md={6} mdOffset={3} sm={8} smOffset={2} xs={12}>
-      <Login block={true} data-test-label='landing-big-cta'>
-        Get started (it's free)
-      </Login>
-    </Col>
-  </Row>
-);
+export const Landing = ({ page = 'landing' }) => {
+  const data = useStaticQuery(graphql`
+    query Images {
+      images: allFile(filter: { relativeDirectory: { eq: "testimonials" } }) {
+        nodes {
+          id
+          name
+          childImageSharp {
+            fixed(width: 200) {
+              ...GatsbyImageSharpFixed
+            }
+            fluid {
+              ...GatsbyImageSharpFluid
+            }
+          }
+        }
+      }
+      image: file(relativePath: { eq: "landing.png" }) {
+        id
+        childImageSharp {
+          fixed(width: 555) {
+            ...GatsbyImageSharpFixed
+          }
+        }
+      }
```

Comment:
I see you are using a single fixed image width. Will we not be needing more than one size for various viewports?

IMHO, using `sharp` and `tranformer-sharp` is an overkill here for one size. You might as well use this query to get more sizes and use the correct one as per the view-port. That would be more useful.

Reply:
Using a fluid image might not make sense since we are currently using the image only for large screens( 1200px+). 
So, regardless of the size of the screen, the container will have a certain width and will be centered.

I added some tests and replaced the fixed image with a fluid one so we could explore displaying it for smaller screens in the next iterations.

:shipit:

Reply:
Not sure if I am missing something. 

Why would we be needing to do GraphQL queries to extract and interpolate images using the `sharp` and `transformer-sharp` if we are shipping for only screen size. 

IMHO, It simply make the whole logic pointless? Why not use a optimized `.webp` of the target width?

We are adding cruft here for something which we will explore later.

Reply:
As a heads up, `.webp` is not (yet) supported by Safari. I believe support will be integrated with the release of Safari 14 later this year.

Reply:
@raisedadead, although this pr does not use  gatsby image for multiple image sizes it does use it for image optimization and loading transition. If you still wish to do those manually for this pr, let me know and I would proceed accordingly.

Reply:
Right. Thanks for clarifying. IMHO, this is currently overengineered. Here is a much smaller simpler, vanilla approach given the current context and current requirements:

https://davidwalsh.name/lazyload-image-fade

Reply:
Here is one more that uses intersection observer. Quite neat and eligant: https://medium.com/front-end-weekly/progressive-image-loading-and-intersectionobserver-d0359b5d90cd

Reply:
`sharp` and `transformer-sharp` add build time binaries for interpolating images. 

Its nice when you have a need to generate dozens of variants of dozens of images. The loading and transition is just the cherry on top. If you only need the cherry why bother about the 3 layer cake 😅

Reply:
Here is what you should ideally do:

1. Compress / Optimize the heck out of the images (png is fine). `imagemin` is a good tool.
2. Use one of the vanilla methods to get the UX working. That is fade in or something when loaded.